### DISCUSSION TITLE
New to Github!  R Language

### DISCUSSION THREAD

**Discussion by catonrachel** [community/community#118644] [Programming Help]
### Body

To whom it  may concern,

This is my first time on GitHub and I don’t know how to submit this as a link to pass my R Certification Lab.  Can anyone with experience respond to help me out?

Best Regards,

~ Rachel C.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @catonrachel, we're happy you're here! 

You are more likely to get a useful response if you are explicit about what you are trying to accomplish. Giving a few more details might help someone give you a nudge in the right direction.
